faker/providers/person/pl_PL/__init__.py

```diff
@@ -774,3 +774,798 @@ def pwz_nurse(self, kind='nurse'):
+        '942',  # Drugi Urząd Skarbowy w Gdyni
+        '943',  # Drugi Urząd Skarbowy w Kielcach
+        '944',  # Urząd Skarbowy Kraków-Dębniki
+        '945',  # Urząd Skarbowy Kraków-Prądnik
+        '946',  # Trzeci Urząd Skarbowy w Lublinie
+        '947',  # Drugi Urząd Skarbowy Łódź-Bałuty
+        '948',  # Drugi Urząd Skarbowy w Radomiu
+        '949',  # Drugi Urząd Skarbowy w Częstochowie
+        '951',  # Urząd Skarbowy Warszawa-Ursynów
+        '952',  # Urząd Skarbowy Warszawa-Wawer
+        '953',  # Drugi Urząd Skarbowy w Bydgoszczy
+        '954',  # Drugi Urząd Skarbowy w Katowicach
+        '955',  # Trzeci Urząd Skarbowy w Szczecinie
+        '956',  # Pierwszy Urząd Skarbowy w Toruniu
+        '957',  # Trzeci Urząd Skarbowy w Gdańsku
+        '958',  # Drugi Urząd Skarbowy w Gdyni
+        '959',  # Drugi Urząd Skarbowy w Kielcach
+        '961',  # Urząd Skarbowy Kraków-Dębniki
+        '962',  # Urząd Skarbowy Kraków-Prądnik
+        '963',  # Trzeci Urząd Skarbowy w Lublinie
+        '964',  # Drugi Urząd Skarbowy Łódź-Bałuty
+        '965',  # Drugi Urząd Skarbowy w Radomiu
+        '966',  # Drugi Urząd Skarbowy w Białymstoku
+        '967',  # Trzeci Urząd Skarbowy w Bydgoszczy
+        '968',  # Drugi Urząd Skarbowy w Kaliszu
+        '969',  # Drugi Urząd Skarbowy w Gliwicach
+        '971',  # Urząd Skarbowy w Gostyninie
+        '972',  # Urząd Skarbowy Poznań-Winogrady
+        '973',  # Drugi Urząd Skarbowy w Zielonej-Górze
+        '974',  # Drugi Urząd Skarbowy w Białymstoku
+        '975',  # Trzeci Urząd Skarbowy w Bydgoszczy
+        '976',  # Drugi Urząd Skarbowy w Bydgoszczy
+        '977',  # Drugi Urząd Skarbowy w Gliwicach
+        '978',  # Urząd Skarbowy w Gostyninie
+        '979',  # Urząd Skarbowy Poznań-Winogrady
+        '981',  # Drugi Urząd Skarbowy w Zielonej Górze
+        '982',  # Drugi Urząd Skarbowy Łódź-Górna
+        '983',  # Drugi Urząd Skarbowy Łódź-Górna
+        '984',  # Urząd Skarbowy w Nidzicy
+        '985',  # Urząd Skarbowy w Nidzicy
+        '986',  # Urząd Skarbowy w Kamieniu Pomorskim
+        '987',  # Urząd Skarbowy w Kamieniu Pomorskim
+        '988',  # Urząd Skarbowy w Wołowie
+        '989',  # Urząd Skarbowy w Wołowie
+        '991',  # Drugi Urząd Skarbowy w Opolu
+        '992',  # Drugi Urząd Skarbowy w Opolu
+        '993',  # Drugi Urząd Skarbowy w Tarnowie
+        '994',  # Drugi Urząd Skarbowy w Tarnowie
+        '995',  # Urząd Skarbowy w Grodzisku Wielkopolskim
+        '996',  # Urząd Skarbowy w Grodzisku Wielkopolskim
+        '997',  # Urząd Skarbowy Wieluń lokalizacja w Wieruszowie
+        '998',  # Urząd Skarbowy Wieluń lokalizacja w Wieruszowie
+    )
+
+    def nip(self):
+        """
+        Returns 10 digit of Number of tax identification.
+        Polish: Numer identyfikacji podatkowej (NIP).
+
+        https://pl.wikipedia.org/wiki/NIP
+        """
+
+        nip = [int(i) for i in self.random_choices(self.tax_office_codes, 1)[0]]
```

Comment:
The call to `self.random_choices(self.tax_office_codes, 1)[0]` can be replaced by something simpler: `self.random_element(self.tax_office_codes)`.

Reply:
change done